### DISCUSSION TITLE
Copilot Subscription Issue

### DISCUSSION THREAD

**Discussion by A9074o** [community/community#129304] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I subscribed to Copilot in April 2024 for $100, which was supposed to cover a full year. Initially, I used the trial version and found it very helpful, so I decided to pay for a one-year subscription. However, by June 2024, my access was frequently interrupted due to attempted charges to my account. Since there were no additional funds in my account, my subscription was suspended.

This situation is very frustrating.

![image](https://github.com/community/community/assets/45235142/6913b2d9-6ff0-4ab8-9999-7b31511b68e3)
![image](https://github.com/community/community/assets/45235142/9993cad7-e3e2-41c0-b340-43e5a80c4eab)
![image](https://github.com/community/community/assets/45235142/2300453a-cc41-4003-9297-f0d4a144ca1f)

**Comment by A9074o** [comment]
I urgently need help with this issue.

**Comment by davevad93** [comment]
Hi @A9074o ,

I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Billing, payments or receipts` and describe your issue. Please note that GitHub may take a while to reply, so be patient.

**Reply by A9074o** [reply]
I have already submitted a ticket 3 days ago, but no one has responded yet. This makes me very upset.

**Reply by davevad93** [reply]
I feel your frustration, support can take a long time to reply indeed. I hope they will reach out to you as soon as possible.

**Reply by A9074o** [reply]
I need a faster solution to resolve this issue. Are there any better and quicker ways to solve it?

**Reply by davevad93** [reply]
As far as I know, no. Opening a discussion in the Community and rasing a ticket to support are the only ways to contact with GitHub.

**Reply by DaydreamCoding** [reply]
This is a bandit. You can cancel my subscription and give me my money back

**Comment by Mee-meet** [comment]
Subject: Request for Unsuspension of GitHub Account

Dear GitHub Support Team,

I hope this message finds you well. My name is Meet Mehta, and I am writing to request the unsuspension of my GitHub account. The details of my account are as follows:

Username: MeetInCode
Email: meets.studentlife@gmail.com
I recently discovered that my account has been suspended, and I am unsure of the reason for this action. I rely heavily on GitHub for my studies and projects, and losing access to my repositories has caused significant disruption.

If the suspension was due to a misunderstanding or an error on my part, I sincerely apologize and am willing to take any necessary corrective actions to ensure compliance with GitHub's terms of service.

I would greatly appreciate your assistance in reviewing my case and providing guidance on how to resolve this issue. Thank you for your understanding and support.

Looking forward to your prompt response.

Best regards,
Meet Mehta
